libpkg/private/utils.h

```diff
@@ -57,8 +57,8 @@ struct hardlink {
 };
 
 struct tempdir {
-	char name[MAXHOSTNAMELEN];
-	char temp[MAXHOSTNAMELEN];
+	char name[MAXPATHLEN];
```

Comment:
Shouldn't it be `PATH_MAX`?

Reply:
I used MAXPATHLEN following what I'd seen in other path vars like in pkg_add_dir_to_del although it seems both are used throughout the codebase. I'm happy to change it.